cluster/client.py

```diff
@@ -127,9 +127,17 @@ def main():
         '-t', '--timeout',
         type=int,
         default=cluster.DEFAULT_TIMEOUT,
-        help='Time in second to let a chance to deploy the service before'
+        help='Time in second to let a chance to deploy the service before '
              'raising an exception (ignored with ``--no-wait`` option)'
     )
+    parser_migrate.add_argument(
+        '-u', '--update',
+        default=True,
```

Comment:
do you mind to use ``action='store_true'```and rename the param as something like ``--migrate-only`` or ``--without-update`` to avoid manage -u [1|0] syntax ?

Reply:
yep, i was initially thinking for ci integration purpose, to have things such as migrate ... --u $UPDATE, so the CI var would be explicitly 0 or 1. so an optional argument with value (no action=store_true). but yes command line is less elegant.

with the action=store_true (argument as option) command line is more elegant, and ci integration perhaps less: migrate $NO_UPDATE, and NO_UPDATE such as "--no_update" or "".

if we keep the 'no update flag approach', --no-update would be great,  i am considering this to focus first with command line ease of use.

Reply:
--no-update flag pushed